packages/web/app/views/patients/panes/PatientProgramsPane.jsx

```diff
@@ -31,6 +32,7 @@ export const PatientProgramsPane = React.memo(({ endpoint, patient }) => {
         patient={patient}
         data-testid="datafetchingprogramstable-uytn"
       />
+      <PatientSurveyAssignmentsTable patient={patient} />
```

Comment:
Looks like there is some work to do to bring this up to the designs but I assume that is in a later ticket?